plugins/@grouparoo/app-templates/public/templates/table-source/{{{id}}}.js.template

```diff
@@ -12,7 +12,8 @@ exports.default = async function buildConfig() {
       type: "{{{__pluginName}}}-table-import",
       appId: {{{appId}}}, // Set this value to the ID of the App this Source uses - e.g. `appId: "data_warehouse"`
       options: {
-        table: {{#from}}{{{from}}}{{/from}}{{^from}}"..."{{/from}}, // Name of the table in your DB - e.g. `table: "users"`
+        table: {{#from}}{{{from}}}
+{{/from}}{{^from}}"..."{{/from}}, // Name of the table in your DB - e.g. `table: "users"`
```

Comment:
The spacing was changed here... what does this look like when mustached?

Reply:
There are some other similar changes below

Reply:
Good catch! This is Prettier or VS Code formatter doing something weird on save.